compat/maven-plugin-api/src/main/java/org/apache/maven/plugin/AbstractMojo.java

```diff
@@ -176,6 +178,17 @@ public Log getLog() {
         return log;
     }
 
+    protected void createFile(File file) throws MojoExecutionException {
```

Comment:
Let's try not to add new things to the compat package. This could easily be a utility method somewhere else, preferably not in src/main since these changes only really touch integrations tests.